opennms-config/src/test/resources/poller-configuration-valid1.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0"?>
```

Comment:
Can we reduce this to a minimum configuration that is required for the test?

Reply:
done